pharmacy-ui/src/views/Home/Home.scss

```diff
@@ -1,38 +1,51 @@
 @import "../../assets/colors.scss";
  
+body {
+  background: rgb(43,149,213);
+  background: linear-gradient(194deg, rgba(43,149,213,0.9809173669467787) 48%, rgba(43,124,154,1) 68%, rgba(58,155,232,0.9332983193277311) 100%);
+}
+
   .wrapper {
     display: grid;
-    grid-template-columns: repeat(2, 1fr);
-    grid-template-areas: 'pharmacy'                       
+    grid-template-columns: [first] 550px [line2] 500px [line3] ;// repeat(2, 1fr);
+    grid-template-areas: 'pharmacy pharmacyDrugDemand'                       
                           'pharmacyCard pharmacist'; 
-    //grid-auto-rows: 100px;
   }
   
   .box1 {
```

Comment:
should be able to replace all these with `grid-area: pharmacy` etc.

Reply:
Done!